src/NuGet.Services.AzureSearch/Registration/IRegistrationClient.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace NuGet.Protocol.Registration
```

Comment:
>NuGet.Protocol.Registration [](start = 10, length = 27)

What do you think of renaming `NuGet.Protocol.Catalog` to `NuGet.Protocol` and moving the registration libraries there?

Reply:
Oh right I forgot about that client package. Yeah I guess there's no need for this right now, ignore my comment.